source/wp-content/themes/wporg-developer-2023/src/search-filters/style.scss

```diff
@@ -1,43 +1,22 @@
 .wp-block-wporg-search-filters {
-	display: grid;
-	margin-top: var(--wp--preset--spacing--10);
-	font-size: var(--wp--preset--font-size--small);
-	grid-template-columns: 1fr 1fr;
-	gap: var(--wp--preset--spacing--10);
-}
-
-.wp-block-wporg-search-filters > *:first-child {
-	grid-column: 1/3;
-}
-
-.wp-block-wporg-search-filters > * {
 	display: flex;
+	flex-wrap: wrap;
 	align-items: center;
-}
+	gap: var(--wp--preset--spacing--20);
+	margin-top: var(--wp--preset--spacing--20);
 
-.wp-block-wporg-search-filters input[type="checkbox"] + label {
-	margin-left: 2px;
-}
-
-@media (min-width: 500px) {
-	.wp-block-wporg-search-filters {
-		display: flex;
-		align-items: center;
+	input[type="checkbox"] {
+		border: 0;
+		clip: rect(1px, 1px, 1px, 1px);
+		clip-path: inset(50%);
+		height: 1px;
+		margin: -1px;
+		overflow: hidden;
+		padding: 0;
+		position: absolute;
+		top: 0;
+		left: 0;
```

Comment:
Can you add a position relative to the container element here? the toggle is at the top of the page, so if you're scrolled down a little tabbing into the checkbox moves the page back up. You can also see the strange highlight in VoiceOver.

<img width="1068" alt="Screenshot 2023-10-03 at 5 57 43 PM" src="https://github.com/WordPress/wporg-developer/assets/541093/6ab90b4f-fc84-42bd-895c-f4456a1cda58">

Reply:
Added to the [toggle styles](https://github.com/WordPress/wporg-parent-2021/pull/112/files#diff-01daabce597632cf10fb204a48422d91c0dfa480fd49f22cb0324afdb0b32a5cR23)